api/Request.json

```diff
@@ -1258,10 +1258,10 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "10.1"
```

Comment:
This is the "navigate_mode" entry. I've filed https://github.com/mdn/browser-compat-data/issues/6580 about why it's not shown on MDN.